packages/core/src/components/Menu/MenuItem/MenuItem.tsx

```diff
@@ -251,14 +248,7 @@ const MenuItem: VibeComponent<MenuItemProps | MenuItemTitleComponentProps> & {
         </div>
       ) : (
         <div className={styles.subMenuIconWrapper}>
-          <Icon
-            clickable={false}
-            icon={DropdownChevronRight}
-            iconLabel={iconLabel}
```

Comment:
Though it's not clickable, it has a functionality, it's the arrow for the submenu, I think it should remain since it will imply that it opens the submenu. So I think that the aria-label should remain nevertheless